test/unit/spec/IDPDiscovery_spec.js

```diff
@@ -92,6 +92,17 @@ function (_, $, Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, IDPDiscoveryF
     .then(function () { return test; });
   }
 
+  function waitForWebfingerCall(test) {
+    return tick() //make webfinger call
+    .then(function () { return test; });
+  }
+
+  function waitForWebfingerResponse(test) {
```

Comment:
can we wait for something else? like ```waitForSpyCall(test.successSpy);``` and when it is okta:idp we might be able to ```waitForPrimaryAuth```?
Just trying to avoid ```tick```